tasks/main.yml

```diff
@@ -2,13 +2,18 @@
 # tasks file for docker
 
 - name: Import install_debian.yml if OS distribution is Debian
-  import_tasks: install_debian.yml
-  when: ansible_distribution == "Debian"
+  import_tasks: user_group.yml
+  when: >
+    ansible_distribution == "Debian" or
+    ansible_distribution == "Ubuntu" or 
```

Comment:
@blmhemu - There's a trailing whitespace at the end after the "**or**" `ansible_distribution == "Ubuntu" or ` , can I please request to remove it? It's just because it gets reported by ansible-lint

```shell
--> Test matrix
    
└── default
    ├── lint
    ├── destroy
    ├── syntax
    ├── create
    ├── converge
    ├── idempotence
    ├── verify
    └── destroy
    
--> Scenario: 'default'
--> Action: 'lint'
--> Executing: yamllint .
ansible-lint
flake8
./tasks/main.yml
  8:40      error    trailing spaces  (trailing-spaces)

```